redis_session.c

```diff
@@ -361,9 +530,14 @@ PS_READ_FUNC(redis)
 #else
     resp = redis_session_key(rpm, ZSTR_VAL(key), ZSTR_LEN(key), &resp_len);
 #endif
-    cmd_len = REDIS_SPPRINTF(&cmd, "GET", "s", resp, resp_len);
+    pool->lock_status->session_key = (char *) emalloc(resp_len + 1);
+    memset(pool->lock_status->session_key, 0, resp_len + 1);
+    strncpy(pool->lock_status->session_key, resp, resp_len);
 
+    cmd_len = REDIS_SPPRINTF(&cmd, "GET", "s", resp, resp_len);
     efree(resp);
+
+    lock_acquire(redis_sock, pool->lock_status TSRMLS_CC);
```

Comment:
Why do you ignore result of `lock_acquire`?

Reply:
Because it does not matter for the current logic.
But I think it makes sense to throw a warning.

In the current logic, a failing lock_aqauire (e.g. redis.session.lock_retries reached) will allow reads but will deny writes.
Do you think we should change this?